incubator/virtualcluster/cmd/syncer/app/options/options.go

```diff
@@ -98,6 +100,7 @@ func (o *ResourceSyncerOptions) Flags() cliflag.NamedFlagSets {
 	fs := fss.FlagSet("server")
 	fs.StringVar(&o.SuperMaster, "super-master", o.SuperMaster, "The address of the super master Kubernetes API server (overrides any value in super-master-kubeconfig).")
 	fs.StringVar(&o.ComponentConfig.ClientConnection.Kubeconfig, "super-master-kubeconfig", o.ComponentConfig.ClientConnection.Kubeconfig, "Path to kubeconfig file with authorization and master location information.")
+	fs.StringVar(&ComponentName, "componentname", ComponentName, "The component name (default vc-syncer).")
```

Comment:
We should add this as a field on the `ResourceSyncerOptions` instead of using a global `var`